pkg/providers/imagefamily/bootstrap/containerdfornbcontract.toml.gtpl

```diff
@@ -0,0 +1,117 @@
+version = 2
+oom_score = 0
+[plugins."io.containerd.grpc.v1.cri"]
+  sandbox_image = "mcr.microsoft.com/oss/kubernetes/pause:3.6"
+  [plugins."io.containerd.grpc.v1.cri".containerd]
+    {{- if .TeleportConfig.GetStatus }}
+    snapshotter = "teleportd"
+    disable_snapshot_annotations = false
+    {{- else}}
+      {{- if .GetIsKata }}
```

Comment:
GetIsKata is verbose why do we do it this way? Why not just 
{{- if .Kata }}

Reply:
gtpl files will be hosted in the go binary on VHD. We will remove them from Karpenter once the Go binary is ready.
And the Getters GetXX is auto generated by protobuf3. It has some benefits like nil-safe. So we widely use the getters unless the variables are optional.